packages/relay/src/lib/relay.ts

```diff
@@ -288,9 +280,9 @@ export class Relay {
         // Invoked when the registry collects its metrics' values.
         // Allows for updated account balance tracking
         try {
-          const operatorAccountId = clientMain.operatorAccountId!.toString();
+          const accountId = operatorAccountId!.toString();
```

Comment:
Why are we using the ! operator here when operatorAccountId is `AccountId | null`?